Bottom: 551c5105e7220fa825fba39c4050d48f4906a8b1
Top:    bc46c5aa735efb013e139457fb87fc695a5b969e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 10:34:57 +0000

Try debugging draw speed




---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index fbd9ecf..b3c525d 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1568,7 +1568,7 @@ Benchmark (int argc, char **argv, Coord x, Coord y)
     }
   while (end - start < 10);
 
-  printf (_("%g redraws per second\n"), i / 10.0);
+  printf (_("%g redraws per second\n"), (double)i / (double)(end-start));
 
   return 0;
 }
